apps/offlineevents/views.py

```diff
@@ -10,3 +19,102 @@ class OfflineEventDetailView(module_views.ItemDetailView):
     @property
     def project(self):
         return self.get_object().project
+
+
+class OfflineEventMgmtView(DashboardBaseMixin,
```

Comment:
Removed the mgmt naming.

With regards to the URLs. I agree with your suggestion, but would fix it with the dashboard  refactoring that anyway will have to define an URL schema.